src/Model/Job.php

```diff
@@ -24,7 +24,8 @@ class Job
     public string $place;
     public ?string $formation;
     public ?string $experience;
-    public ?string $remuneration;
+    public ?int $remunerationMin;
+    public ?int $remunerationMax;
```

Comment:
Pour éviter d'avoir des erreurs d'accès à cette propriété non-initialisée lorsque le contenu ne précise pas du tout la propriété, fournir une valeur par défaut à null. Ainsi la propriété est toujours initialisée :  

```suggestion
    public ?int $remunerationMin = null;
    public ?int $remunerationMax = null;
```